src/main/java/randoop/sequence/Variable.java

```diff
@@ -84,11 +84,7 @@ public Variable copyWithIndexUpdated(Map<Integer, Integer> map) {
   }
 
   public static List<Integer> statementIndexList(List<Variable> values) {
-    List<Integer> result = new ArrayList<>(values.size());
-    for (Variable value : values) {
-      result.add(value.getDeclIndex());
-    }
-    return result;
+    return CollectionsPlume.mapList(Variable::getDeclIndex, values);
```

Comment:
Why no "import org.plumelib.util.CollectionsPlume" in this file?  If not needed, how is it different for others?

Reply:
This particular change is within a comment (see line 93/89 below, or expand the code view upward to see the opening comment).